net/seafile-seahub/Makefile

```diff
@@ -8,67 +8,86 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=seafile-seahub
-PKG_VERSION:=6.3.4
-PKG_RELEASE:=8
+PKG_VERSION:=7.1.2
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/haiwen/seahub/tar.gz/v$(PKG_VERSION)-server?
-PKG_HASH:=53a9efdb6791fd3a2a191e89cb0f133632056046ec08adbb2ad72088e6161430
+PKG_HASH:=647dca870c996ed00844d3bf45bc15f22fdf5268cfd3ac4c0dd9e4c15360361d
 
-PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
+PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>, Jeffery To <jeffery.to@gmail.com>
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE.txt
 
-HOST_PYTHON_PACKAGE_BUILD_DEPENDS:="django>=1.11"
-
 PKG_BUILD_DIR:=$(BUILD_DIR)/seahub-$(PKG_VERSION)-server
 
-include $(INCLUDE_DIR)/package.mk
-include ../../lang/python/python-package.mk
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:="Django~=1.11"
 
-SEAFILE_PYTHON_DEPENDS:= \
-	simplejson chardet dateutil mysqlclient pytz qrcode requests requests-oauthlib \
-	django1 django-constance django-appconf django-compressor django-formtools \
-	django-jsonfield django-picklefield django-postoffice django-restframework39 \
-	pillow django-simple-captcha django-statici18n django-webpack-loader
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
 
 define Package/seafile-seahub
   SECTION:=net
   CATEGORY:=Network
   TITLE:=Seafile server - seahub component
   URL:=https://seafile.com/
-  DEPENDS:=+python \
-	+python-flup +gunicorn +openpyxl \
-	$(foreach dep,$(SEAFILE_PYTHON_DEPENDS),+python-$(dep))
+  DEPENDS:= \
+    +gunicorn3 \
+    +python3 \
+    +python3-chardet \
```

Comment:
why not keep the `foreach`?

Reply:
The `foreach` loop saves the initial, one-time cost of typing the `python3-` prefix. The ongoing, cognitive cost is when you have to work with / think about the package dependencies:

* Remember that this package is different that every other package; read and understand the code to see what it does
* Think about whether the package name you are considering has a `python3-` prefix, and mentally remove it if it does
* Choose one of two lists of dependencies, depending on whether the package name has the `python3-` prefix or not

I don't think the benefit outweighs the cost in this case.